src/Lykke.Service.ExchangeConnector/Exchanges/Abstractions/Exchange.cs

```diff
@@ -20,9 +20,9 @@ internal abstract class Exchange : IExchange
 
         private readonly List<Handler<OrderBook>> _orderBookHandlers = new List<Handler<OrderBook>>();
 
-        private readonly List<Handler<ExecutedTrade>> _executedTradeHandlers = new List<Handler<ExecutedTrade>>();
+        private readonly List<Handler<OrderStatusUpdate>> _executedTradeHandlers = new List<Handler<OrderStatusUpdate>>();
```

Comment:
We should consider using only one type of handlers now, just to handle an update on the order's status. It can be triggered in both cases, for execution of the order and for an acknowledgment after placing an order to the order book.